src/table-theme-colors.js

```diff
@@ -1,8 +1,23 @@
-import { isDarkColor } from './table/utils/color-utils';
+import { isDarkColor, isTransparentColor } from './table/utils/color-utils';
 
 export default function tableThemeColors(theme) {
-  const backgroundColor = theme.getStyle('object', 'straightTable', 'backgroundColor');
-  const isBackgroundDarkColor = isDarkColor(backgroundColor);
+  const qvInnerObject = document.querySelector('.qv-object .qv-inner-object');
+  const objectBackgroundColorFromCSS = qvInnerObject && window.getComputedStyle(qvInnerObject).backgroundColor;
+  const opacityFromCSS = qvInnerObject && window.getComputedStyle(qvInnerObject).opacity;
+  const qvPanelSheet = document.querySelector('.qv-panel-sheet');
+  const sheetBackgroundColorFromCSS = qvPanelSheet && window.getComputedStyle(qvPanelSheet).backgroundColor;
+
+  const tableBackgroundColorFromTheme = theme.getStyle('', '', 'object.straightTable.backgroundColor');
+  const backgroundColorFromTheme = theme.getStyle('object', 'straightTable', 'backgroundColor');
```

Comment:
ok more questions... Isn't this already applied to `qvInnerObject`? And what happens outside the client? do we have access to the theme at all?

Reply:
inside the client, yes, it gets the same backgroundColor. But in the mashup, we can only get the backgroundColor from the theme, not from the qvInnerObject.
Yes, we do have access to the theme https://github.com/qlik-oss/sn-table/blob/main/mashup-example/using-build-tool/qlik-sense-desktop/configure.js#L17

Reply:
ok that makes sense. The only thing I am missing now is some tests on this file

Reply:
this isn't resolved. I don't see any tests on this file at all. Please add some, since this logic is not that straight forward

Reply:
do you mean unit tests for table-theme-colors.js?

Reply:
yes

Reply:
okay, gonna try to add some

Reply:
Done